app/javascript/components/applications/application_form/AlternateContactSection.js

```diff
@@ -1,23 +1,39 @@
 import React from 'react'
-import { isEmpty } from 'lodash'
+import { isEmpty, values } from 'lodash'
 import { Form, NestedForm, Text, Select } from 'react-form'
 import formOptions from './formOptions'
 import AddressForm from './AddressForm'
 import { mailingAddressFieldMap } from './utils'
+import validate from '~/utils/form/validations'
+import { Field } from '~/utils/form/Field'
 
 const {
   alternateContactOptions,
   phoneTypeOptions
 } = formOptions
 
+const validateError = (formValues) => {
+  return {
```

Comment:
the way we do it for householdMember validations, is to check if it's empty before doing the validations. 
https://github.com/Exygy/sf-dahlia-lap/blob/master/app/javascript/components/applications/application_form/PaperApplicationForm.js#L63-L74. I think if you did a similar thing here that would get rid of the need for `validateFirstLastName`

Reply:
Yes, I initially did this but the formApi wouldn’t get updated when I tried to clear the empty string. I can try again and get a 2nd pair of eyes on it

Reply:
@akegan I implemented this way and looks like it works -- modifying formValues directly than accessing the formApi :+1:

Reply:
hm, for this I meant that you could replace validateFirstLastName with just validate.isPresent. Let me know if you want to hop on a quick call about it